src/App.tsx

```diff
@@ -90,8 +90,12 @@ const App: React.FC = () => {
   }, [dispatch, user.id]);
 
   useEffect(() => {
-    user.id && (!user.authority || !user.orcid) && setShowAuthorityOrcidModal(true);
-  }, [user.id, user.authority, user.orcid]);
+    setTimeout(() => {
+      user.id &&
+        (user.authority?.orcids.length === 0 || user.authority?.feideids.length === 0) &&
+        setShowAuthorityOrcidModal(true);
+    }, DEBOUNCE_INTERVAL_MODAL);
```

Comment:
hva var forklaringen på denne?

Reply:
det er satt pga det tar ganske lang tid å få svar fra BARE-proxyen, og det er dumt at modalen vises (flashes) for så å bli lukket rett etterpå. Burde lage en loader-komponent som fikser dette etterhvert

Reply:
Ser det er flere kall som tar forholdsvis lang tid. Kanskje popup/blokkering av applikasjone er det beste ?

Reply:
hva mener du med blokkering av applikasjonene?